__tests__/app.test.js

```diff
@@ -263,6 +264,41 @@ describe("POST /api/articles/:article_id/comments",() => {
   });
 });
 
+describe("GET /api/articles (topic query)",() => {
+  it("200: Should filter the articles by topic specified in the query.", () => {
+    return request(app) // send request to app.
+      .get("/api/articles?topic=cats") //GET req to endpoint.
+      .expect(200)
+      .then(({ body }) => {
+        const { articlesWithTopic } = body; //deconstruct articles from body.
+        expect(articlesWithTopic).toHaveLength(1); //length check
+        expect(articlesWithTopic).toEqual(
+          [
+            {
+              article_id: 5,
+              title: 'UNCOVERED: catspiracy to bring down democracy',
+              topic: 'cats',
+              author: 'rogersop',
+              body: 'Bastet walks amongst us, and the cats are taking arms!',
+              created_at: '2020-08-03T13:14:00.000Z',
+              votes: 0,
+              article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+            }
+          ]
+        )
```

Comment:
We'll need a couple more tests here please:
- 200 topic with multiple articles
- 200 topic which exists but has 0 articles